tables/output_tables.Rmd

```diff
@@ -76,18 +76,86 @@ histology_df %>%
+"Kids_First_Participant_ID","KidsFirst patient identifier","PT_########",
+"molecular_subtype","Molecular subtype defined by WHO 2016 guidelines","text",
+"normal_fraction","Theta2 normal DNA fraction estimate","numeric",
+"Notes","Free text field describing changes from `pathology_diagnosis` to `integrated_diagnosis` or manner in which molecular_subtype was determined","text",
+"OS_days","Overall survival in days","numeric",
+"OS_status","Overall survival status",paste(unique(histology_df$OS_status), collapse = "; "),
+"parent_aliquot_id","External identifier combining sample_id; sample_type; aliquot_id; sequencing_strategy for some samples","alphanumeric",
+"pathology_diagnosis","Reported and/or harmonized patient diagnosis from pathology reports","text",
+"pathology_free_text_diagnosis","Free text patient diagnosis from pathology reports","text",
+"PFS_days","Progression-free survival in days","numeric",
+"primary_site","Bodily site(s) from which specimen was derived","text",
+"race","Patient reported race",paste(unique(histology_df$race), collapse = "; "),
+"reported_gender","Patient reported gender",paste(unique(histology_df$reported_gender), collapse = "; "),
+"RNA_library","Type of RNA-Sequencing library preparation",paste(unique(histology_df$RNA_library), collapse = "; "),
+"sample_id","External biospecimen identifier","alphanumeric",
+"sample_type","Broad sample type",paste(unique(histology_df$sample_type), collapse = "; "),
+"seq_center","Sequencing center",paste(unique(histology_df$seq_center), collapse = "; "),
+"short_histology","Abbreviated `integrated_diagnosis` or `broad_histology` for plotting purposes",paste(unique(histology_df$short_histology), collapse = "; "),
+"tumor_descriptor","Phase of therapy from which tumor was derived",paste(unique(histology_df$tumor_descriptor), collapse = "; "),
+"tumor_fraction","Theta2 tumor DNA fraction estimate","numeric",
+"tumor_ploidy","Control-FREEC ploidy","numeric"
+)
+
+# Sheet 2: Define array of samples that need to be classified from [ticket](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1206) which are not in v21, but set to be changed in final release [v22](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1207)
 to_be_classified_samples <- c("BS_3J4T2YYW", "BS_7F07M7JG", "BS_AG0BZN6F", "BS_HZNKSQ17", "BS_V6HKXX1Z", "BS_WP9J88EB", "BS_YYAPSA5P")
 
-histology_df %>% 
+histologies_table <- histology_df %>% 
   # Update definitions of molecular subtype for TBD samples
   mutate(molecular_subtype = ifelse(
     Kids_First_Biospecimen_ID %in% to_be_classified_samples,
     "MB, To be classified",
     molecular_subtype
-  )) %>%
-  # export with argument `keepNA=TRUE` so NAs come up as NAs and not blank cells
-  write.xlsx(table_s1_file, overwrite=TRUE, keepNA=TRUE)
+  ))
+
+# Sheet 3: CNS region definition based on definitions from [Cassie Kline](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1025). 
```

Comment:
I am not seeing a lot of this information in the attached text document. For example, spine, suprasellar,  "other", Posterior fossa. The other link in this PR to the d3b repo is 404.

Is there somewhere else we have this full information for the samples?

Reply:
Ah, this was the manual pathology report inspection and updating of CNS region, which was separate of the logic - let me find that and add it.

Reply:
updated in last commit with JSON assistance from @logstar